Hyperlint Automation: Meta Description Fixes (#20466)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/devices/service-providers/custom.mdx

  label: Custom integration
  order: 1

> description: Configure custom device posture checks in Cloudflare Zero Trust using WARP service-to-service integration. Set API calls for device posture scoring.

Reviewer: ```suggestion
description: Configure custom device posture checks in Cloudflare Zero Trust using a service-to-service integration.
```
